examples/flutter_gallery/test/smoke_test_semantics.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'smoke_test.dart';
+
+void main() {
+  testWidgets('Enable semantics', (WidgetTester tester) async {
+    RendererBinding.instance.setSemanticsEnabled(true);
```

Comment:
in theory nothing guarantees the order the tests run in and they shouldn't depend on each other.

I think the right way to factor this out would be to have the inside of the "Flutter Gallery app smoke test" in its own function, and then have the two tests both call that function. The semantics one should enable the semantics before and disable them after. I think it'd be fine for these two tests to be in the same file (otherwise it's a bit weird because one test imports the other).